src/utils/services/ColorGenerator.ts

```diff
@@ -40,11 +41,12 @@ export default class ColorGenerator {
     return color;
   };
 
-  public readonly isColorLightForKey = (key: string) => {
+  public readonly isColorLightForKey = (key: string): boolean => {
     const colorHex = this.getColorForKey(key).substring(1);
 
-    if (this.lights[colorHex] == undefined) {
+    if (!this.lights[colorHex]) {
```

Comment:
This change breaks the cache optimisation. The `undefined` check was for whether it was calculated for that tag already,  as `false` is a valid value, now it will always recalculate on dark tags.

Reply:
Right! Good catch.

Reply:
This should fix it https://github.com/shlinkio/shlink-web-client/pull/566/files
![image](https://user-images.githubusercontent.com/2719332/148647342-75bc49f0-233c-4746-abb0-7df56a5fb966.png)